src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/CohortDomainServiceTests.cs

```diff
@@ -879,8 +878,7 @@ public void VerifyEmptyCohortCreation(Party party)
             {
                 if (party == Party.Provider)
                 {
-                    Provider.Verify(x => x.CreateCohort(Provider.Object, It.Is<AccountLegalEntity>(p => p == AccountLegalEntity.Object), null,
-                        null, UserInfo));
+                    Provider.Verify(x => x.CreateCohort(Provider.Object, It.Is<AccountLegalEntity>(p => p == AccountLegalEntity.Object), UserInfo));
```

Comment:
this can be simplified to be 

Provider.Verify(x => x.CreateCohort(Provider.Object, AccountLegalEntity.Object, UserInfo));